quicklogic/README.md

```diff
@@ -1,6 +1,8 @@
-# SymbiFlow for QuickLogic FPGAs
+# SymbiFlow for Quicklogic FPGAs
 
-Currently the supported families are:
+Currently, the supported families are:
+
+- EOS-S3
```

Comment:
The family is actually `pp3`. You could change that to eg. " - pp3 (EOS-S3 SoC)"

Reply:
Fixed to `pp3 (EOS-S3 SoC)`